include/deal.II/numerics/data_out_faces.h

```diff
@@ -182,8 +167,8 @@ class DataOutFaces : public DataOut_DoFData<DoFHandlerType,
    * hp::MappingCollection in case of a DoFHandler with hp-capabilities.
    */
   virtual void
-  build_patches(const Mapping<dimension> &mapping,
-                const unsigned int        n_subdivisions = 0);
+  build_patches(const Mapping<dim> &mapping,
+                const unsigned int  n_subdivisions = 0);
```

Comment:
It's probably an oversight that this is not `Mapping<dim,spacedim>`. But that's also for a later patch.